application/namespaces/user/controllers.py

```diff
@@ -55,6 +58,50 @@ def _create_keycloak_client() -> orodha_keycloak.OrodhaKeycloakClient:
     )
 
 
+def get_bulk_users(token: str, payload: dict) -> list:
+    """
+    Function that takes a JWT and optional filters, then returns truncated user data.
+
+    Args:
+        token(str): A JWT token provided by keycloak.
+        payload(dict): The post payload containing filter data and user data
+
+    Raises:
+        OrodhaForbidenError: If the JWT token is not valid or is missing.
+
+    Returns:
+        response_data(list[UserDocument]): A list of matching user documents that have been trimmed
+            to the id and username values only.
+    """
+    keycloak_client = _create_keycloak_client()
+
+    try:
+        keycloak_client.get_user(token=token).get("id")
+    except KeycloakGetError:
+        raise OrodhaForbiddenError(
+            message="You don't have the required token to access this resource."
+        )
+
+    page_size = payload.get("page_size")
```

Comment:
@Edison-Stuart camelCase the parameters that the FE is going to be handling, e.g.

```
page_size = payload.get("pageSize")
```